src/core/client/admin/test/moderate/feature.spec.tsx

```diff
@@ -0,0 +1,129 @@
+import { pureMerge } from "coral-common/utils";
+import {
+  GQLResolver,
+  ModerationQueueToCommentsResolver,
+  QueryToCommentResolver,
+} from "coral-framework/schema";
+import {
+  createQueryResolverStub,
+  createResolversStub,
+  CreateTestRendererParams,
+  replaceHistoryLocation,
+  waitForElement,
+  within,
+} from "coral-framework/testHelpers";
+
+import create from "../create";
+import {
+  emptyModerationQueues,
+  emptyRejectedComments,
+  featuredComments,
+  settings,
+  unmoderatedComments,
+  users,
+} from "../fixtures";
+
+const viewer = users.admins[0];
+
+beforeEach(async () => {
+  replaceHistoryLocation("http://localhost/admin/moderate");
+});
```

Comment:
This location is always replaced to a different location in your tests, maybe we don't need these 3 lines then?

Reply:
Fixed